infra/bridge/src/utils/validation.js

```diff
@@ -19,6 +19,12 @@ const identityValidation = check('identity')
   .withMessage('Field identity must not be empty.')
   .trim()
 
+const identityProxyValidation = check('identityProxy')
```

Comment:
Some legacy users may not have a proxy. Should that field be optional ?

Reply:
Will update it